src/js/components/TextInput/TextInput.js

```diff
@@ -348,7 +354,9 @@ class TextInput extends Component {
     delete rest.onChange; // se we can manage in this.onChange()
     delete rest.onSuggestionsOpen;
     delete rest.onSuggestionsClose;
-    const { showDrop } = this.state;
+    const { showDrop, isInputEmpty } = this.state;
+    const showStyledPlaceholder =
+      placeholder && typeof placeholder !== 'string' && !value && isInputEmpty;
```

Comment:
I think you may not need this conditional `!value` since you already are using `isInputEmpty`

Reply:
make sense!!
tnks man!! :)
i'll fix it